cobalt/dom/captions/system_caption_settings.cc

```diff
@@ -629,6 +641,25 @@ const char* SystemCaptionSettings::CaptionOpacityPercentageToString(
   }
 }
 
+bool SystemCaptionSettings::GetCaptionSettings(
+    SbAccessibilityCaptionSettings* caption_settings) {
+#if SB_API_VERSION >= 16
+  auto platform_info_extension =
+      static_cast<const CobaltExtensionAccessibilityAPI*>(
+          SbSystemGetExtension(kCobaltExtensionAccessibilityName));
+  if (platform_info_extension &&
+      strcmp(platform_info_extension->name,
+             kCobaltExtensionAccessibilityName) == 0 &&
+      platform_info_extension->version >= 1) {
+    return platform_info_extension->GetCaptionSettings(caption_settings);
+  } else {
+    return SbAccessibilityGetCaptionSettings(caption_settings);
+  }
+#else   // SB_API_VERSION >= 16
```

Comment:
Should it be "// SB_API_VERSION < 16" ?

Reply:
No, this looks right to me. Our convention is to use the condition from the `#if` directive for the `#else` and `#endif` directive comments. Please see https://github.com/youtube/cobalt/pull/1930 for example.